### PR TITLE
Updates to CI and plotting

### PR DESCRIPTION
**Pull request by dustinswales**
This PR contains the following changes:
- Update to SCM RT Baseline (Thanks @mkavulich)
- More cleanup of the plotting script.
- New comparison script for offline use (uses the same plotting script as in the RT CI)

### CODE REVIEW

**Comment by dustinswales** [comment]
@grantfirl @mkavulich Can one of you review this?
(I know I need to revert the nml change, I did this just for testing)

**Review by grantfirl — APPROVED** [review]
Looks good except for nml change.

**Comment by dustinswales** [comment]
> Looks good except for nml change.

Done.

File: test/plot_scm_out.py [hunk 1]
```diff
@@ -157,6 +157,7 @@ def plot_results(file_bl, file_rt=None, vars2plt=None):
                     if file_rt is not None: plt.subplot(3,1,1)
                     plt.contourf(x1, y1, z1, 20, cmap='YlGnBu')
                     plt.ylim(1000,200)
+                    plt.xlim(0,np.max(x1))
```
[the remark this answers is not in the document]
Comment (grantfirl) [reply test/plot_scm_out.py:160]: No prob. Ready to merge?

[the remark this answers is not in the document]
Comment (dustinswales) [reply test/plot_scm_out.py:160]: Do it.

**Pull request merged**